Containerfile.image_build

```diff
@@ -69,9 +69,8 @@ ADD image_build/buildah/containers.conf /etc/containers/
 
 # Copy & modify the defaults to provide reference if runtime changes needed.
 # Changes here are required for running with fuse-overlay storage inside container.
-RUN sed -e 's|^#mount_program|mount_program|g' \
-        -e '/additionalimage.*/a "/var/lib/shared",' \
-        -e 's|^mountopt[[:space:]]*=.*$|mountopt = "nodev,fsync=0"|g' \
+RUN sed -e '/additionalimage.*/a "/var/lib/shared",' \
+        -e 's|^mountopt[[:space:]]*=.*$|mountopt = "nodev,metacopy=on"|g' \
```

Comment:
Could you explain the changes a bit more in the commit message? I get that we're no longer uncommenting `#mount_program` and that we're setting a different value for `mountopt`, but I don't quite understand why and what it means.

Reply:
@zxiong could you add that extra info here?

Reply:
Thanks!